src/VisionModel/vgg.jl

```diff
@@ -60,32 +53,37 @@ function classifier_layers(imsize, nclasses, fcsize, dropout)
   return layers
 end
 
-function vgg(imsize; config, batchnorm=false, inchannels=3, nclasses, fcsize=4096, dropout=0.5)
-  conv = convolutional_layers(config, batchnorm, inchannels)
-  class = classifier_layers(imsize, nclasses, fcsize, dropout)
+function vgg(imsize; config, batchnorm=false, nclasses)
+  conv = convolutional_layers(config, batchnorm, 3)
+  class = classifier_layers(imsize, nclasses, 4096, 0.5)
   return Chain(conv..., class...)
 end
 
-vgg11(imsize; inchannels=3, nclasses, fcsize=4096, dropout=0.5) =
-  vgg(imsize, config=:A, inchannels=inchannels, nclasses=nclasses, fcsize=fcsize, dropout=dropout)
+const configs = Dict(:A => [(64,1) (128,1) (256,2) (512,2) (512,2)],
+                     :B => [(64,2) (128,2) (256,2) (512,2) (512,2)],
+                     :D => [(64,2) (128,2) (256,3) (512,3) (512,3)],
+                     :E => [(64,2) (128,2) (256,4) (512,4) (512,4)])
+
+vgg11(imsize; nclasses) =
+  vgg(imsize, config=:A, nclasses=nclasses)
```

Comment:
Now that we are modifying this, can you adjust this so that `config` is passed in as an array of tuples? So instead of
```julia
vgg(imsize, config = :A, ...)
```
we do
```julia
vgg(imsize, config = configs[:A], ...)
```

----

(Oddly, I cannot comment on lines that you did not edit).

On L31, can you change it from
```julia
for c in configs[config]
```
to
```julia
for c in config
```

These two changes will allow us to still use the standard config dictionary, but if someone wants to pass in a custom array of tuples, then they can.